### PR TITLE
GitIssue-903 : Fixes in QuickStartGuide

### PR DESCRIPTION
**Pull request by hessio**
Signed-off-by: Patrick Hession <patrick.hession@seagate.com>

Tested with: VMWare Workstation 16 Pro

Resolves issue #903 
-----
[View rendered docs/CORTX-S3 Server Quick Start Guide.md](https://github.com/Seagate/cortx-s3server/blob/hessio-patch-1/docs/CORTX-S3 Server Quick Start Guide.md)

### CODE REVIEW

**Review by sachitanands — APPROVED** [review]
looks good

File: docs/CORTX-S3 Server Quick Start Guide.md [hunk 1]
```diff
@@ -163,7 +153,7 @@ Before your test your build, ensure that you have installed and configured the f
 2. Ensure you've installed pip.
     - To check if you have pip installed, run the command: `$ pip --version`
     - To install pip, run the command: `$ easy_install pip`
-3. If you don't have Python Version 2.6.5+, install Python using: `$ yum install python26`
+3. If you don't have Python Version 2.6.5+, install Python using: `$ yum install python`
```
Comment (amitwac2608) [inline docs/CORTX-S3 Server Quick Start Guide.md:156]: @sachitanands I guess we no longer using python 2.x, we have moved to python 3.x, right ?

**Comment by hessio** [comment]
Done. @nileshgovande  thank you for this.

File: docs/CORTX-S3 Server Quick Start Guide.md [hunk 2]
```diff
@@ -1,353 +1,345 @@
...
+        * To check if epel is installed, use: `$ yum repolist`
+            * If epel was installed, you'll see it in the output list.
+            * You might also see exclamation mark in front of the repositories id. Refer to the [Redhat Knowledge Base](https://access.redhat.com/solutions/2267871).
+        * `$ yum install -y epel-release`
+
+6. You'll need to install CORTX Python Utilities. Follow the steps to install [CORTX Python Utilities](https://github.com/Seagate/cortx-utils/blob/main/py-utils/README.md).
+
+7. You'll need to install Kafka Server. Follow the steps to install [Kafka Server](https://github.com/Seagate/cortx-utils/wiki/Kafka-Server-Setup).
+
+8. You will need to set your hostname to something other than localhost `hostnamectl set-hostname --static --transient --pretty <new-name>`
```
Comment (nileshgovande) [inline docs/CORTX-S3 Server Quick Start Guide.md:52]: Why is this needed @hessio  ?

**Review by nileshgovande — APPROVED** [review]
